ui/src/app/user/user.component.html

```diff
@@ -82,8 +82,7 @@
             <ion-item lines="none">
               <ion-button style="padding-top: 0.3em" slot="start" (click)="enableAndDisableEditMode()"
                 [disabled]="!isEditModeDisabled && !this.form?.formGroup?.dirty">
-                <ion-label>{{isEditModeDisabled ? ('General.EDIT' | translate) :('General.RESET' |
-                  translate)}}</ion-label>
+                <ion-label>{{ getEditButtonText() | translate }}</ion-label>
```

Comment:
Calling functions from an Angular HTML template should be avoided for performance reasons. Instead the `getEditButtonText()` method seems to be left over and should be deleted.